### PR TITLE
module env changes

### PR DESCRIPTION
**Pull request by mand35**
- removing the module purge suggestion, since we do not recommend that anymore.
- adding short description on how to write own modules

### CODE REVIEW

File: _lessons/maui-and-mahuika/06-software-modules.md [hunk 1]
```diff
@@ -68,19 +68,47 @@ more powerful search capability. The `ml` shortcut can be used in place of
...
+```
+#%module
+
+conflict MyProductionEvn
+module load gimkl
+module load forge
+module load perftools-base perftools
+
+setenv CFLAGS "-DXYZDEBUG "
+prepand-path PATH ~/tools/bin
```
Comment (chrisdjscott) [inline _lessons/maui-and-mahuika/06-software-modules.md]: `prepend-path`?

File: _lessons/maui-and-mahuika/06-software-modules.md [hunk 2]
```diff
@@ -68,19 +68,47 @@ more powerful search capability. The `ml` shortcut can be used in place of
...
+```
+
+To make that directory available to the module environment you need to specify:
+```
+module use ~/modulefiles
+```
+which you can also define in your `~/.bashrc`.
+To load that environment, you simply load your module:
+```
+module load MyProductionEvn
```
Comment (chrisdjscott) [inline _lessons/maui-and-mahuika/06-software-modules.md]: Shouldn't this be `module load MyDebugEnv`?

File: _lessons/maui-and-mahuika/06-software-modules.md (hunk 1 above)
Comment (mand35) [reply _lessons/maui-and-mahuika/06-software-modules.md]: What is wrong with prepend-path? You can also use `append-path`, but I prefer to put custom directories at the beginning, thus if executables exist multiple times, my specified one is preferred.

File: _lessons/maui-and-mahuika/06-software-modules.md (hunk 2 above)
Comment (mand35) [reply _lessons/maui-and-mahuika/06-software-modules.md]: yes, right.

File: _lessons/maui-and-mahuika/06-software-modules.md (hunk 1 above)
Comment (chrisdjscott) [reply _lessons/maui-and-mahuika/06-software-modules.md]: The spelling is wrong `prepand-path` -> `prepend-path` 😄

Comment (mand35) [reply _lessons/maui-and-mahuika/06-software-modules.md]: I agree

**Comment by chrisdjscott** [comment]
Looks good, thanks

**Pull request merged**